content/20.treatments.md

```diff
@@ -147,19 +147,47 @@ Summarize the state of the antiviral approach.
 
 #### Protease Inhibitors
 
-Why it may be useful
+The inhibition of virus proteases could be a promising approach to stop to the replication of the pathgen.
```

Comment:
Hmph. IMO, we should refrain from strong recommendations in a specific section. It may be better to keep each section more evidence-based and evaluate them collectively in the discussion section?

Reply:
Thanks @ypar for reviewing this and catching all the errors.
What would you suggest to have in the `why it may be useful?` section of` Protease Inhibitors`

Reply:
pathgen -> pathogen

Reply:
something along the lines of:

molecular studies showed that viral proteases play an important role in the life cycle of (corona)viruses by modulating the cleavage of viral polyprotein precursors, etc. FDR approved drugs already exist (e.g. HIV, HepC, ...). some evidence shows that a cocktail of protease inhibitors and immune boosters could be beneficial for sars-cov-2. some evidence showed a type of protease inhibitor worked for sars and mers... etc.
but limitations include non-randomized trials used for some of these evaluations, many existing inhibitors are for retrovirals and cov is not.. etc.

Reply:
I tried to develop those ideas in 24b4357. @ypar please let me know if they make sense.